drivers/common/opensensors.h

```diff
@@ -0,0 +1,30 @@
+#ifndef __OPENSENSORS_H__
+#define __OPENSENSORS_H__
+
+#define MAXSENSORS           10
+
+typedef uint16_t (*callbackRead_cbt)(void);
+
+typedef float (*callbackConvert_cbt)(uint16_t value);
+
+typedef struct {
```

Comment:
this is very strange. The drivers must be independent from all application-layer considerations, and cannot for example contain CoAP paths